Trie/Program.cs

```diff
@@ -0,0 +1,119 @@
+﻿TrieNode root;
+Trie trie = new Trie();
+Console.WriteLine("Choose an option:");
+Console.WriteLine("If you want to add the element, enter 1");
+Console.WriteLine("If you want to find the element, enter 2");
+Console.WriteLine("If you want to remove the element, enter 3");
+Console.WriteLine("If you want to know how many elements start with the prefix you entered, enter 4");
+Console.WriteLine("If you want to know how many elements are in a Trie, enter 5.");
+
+int option;
+string enteredWord;
+bool exit;
+do
+{
+    Console.WriteLine("Enter the digit:");
+    enteredWord = Console.ReadLine();
+    if (!int.TryParse(enteredWord, out option))
```

Comment:
Тут можно объявить option
```suggestion
    if (!int.TryParse(enteredWord, out var option))
```